src/map/glmap/Map.js

```diff
@@ -208,6 +217,7 @@ Map.propTypes = {
   transitionEnd: PropTypes.func,
   cursor: PropTypes.string,
   hasHeatmapLayers: PropTypes.bool.isRequired,
+  markers: PropTypes.array,
```

Comment:
Would you consider adding detail here using `arrayOf` ? Or as a rule do we consider ts definitions sufficient?

Reply:
Good catch, added here: https://github.com/GlobalFishingWatch/map-components/commit/6d64577650294bfd172ed42e386d12a3b11ada82

Reply:
added the fix and we can discuss how to manage ts and proptypes definitions later